source/extensions/tracers/skywalking/tracer.cc

```diff
@@ -64,6 +64,17 @@ Tracing::SpanPtr Span::spawnChild(const Tracing::Config&, const std::string& nam
   return std::make_unique<Span>(child_span, tracing_context_, parent_tracer_);
 }
 
+void Span::packSpanContextToMetadata(StreamInfo::StreamInfo& info) const {
+  ProtobufWkt::Struct output;
+  auto* fields = output.mutable_fields();
+  (*fields)["sw_service"] = ValueUtil::stringValue(tracing_context_->service());
+  (*fields)["sw_service_instance"] = ValueUtil::stringValue(tracing_context_->serviceInstance());
```

Comment:
I would suggest to take a value on RHS, and then use it in the following (less typing? :) https://github.com/envoyproxy/envoy/blob/main/source/extensions/filters/network/rocketmq_proxy/protocol.cc#L13-L78
```suggestion
  auto& fields = *output.mutable_fields();
  fields["sw_service"] = ValueUtil::stringValue(tracing_context_->service());
  fields["sw_service_instance"] = ValueUtil::stringValue(tracing_context_->serviceInstance());
```

Reply:
+1